sympy/polys/multivariate_resultants.py

```diff
@@ -0,0 +1,357 @@
+"""
+This module contains functions for two multivariate resultants. These are:
+
+- Dixon's resultant.
+- Maucalay's resultant.
+
+Multivariate resultants are used to identify whether a multivariate system has
+common roots. That is when the resultant is equal to zero.
+"""
+
+from sympy import IndexedBase, Matrix, Mul, Poly, prod, fraction
+from sympy.core.compatibility import range
+from sympy.polys.monomials import monomial_deg
+from sympy.polys.monomials import itermonomials
+from sympy.polys.orderings import monomial_key
+from sympy.polys.polytools import poly_from_expr
+from sympy.functions.combinatorial.factorials import binomial
+
+class DixonResultant():
+    """
+    A class for retrieving the Dixon's resultant of a multivariate system.
+
+    Examples
+    ========
+    >>> from sympy.core import symbols
+
+    >>> from sympy.polys.multivariate_resultants import DixonResultant
+    >>> x, y = symbols('x, y')
+
+    >>> p = x + y
+    >>> q = x ** 2 + y ** 3
+    >>> h = x ** 2 + y
+
+    >>> dixon = DixonResultant(variables=[x, y], polynomials=[p, q, h])
+    >>> poly = dixon.get_dixon_polynomial()
+    >>> matrix = dixon.get_dixon_matrix(polynomial=poly)
+    >>> matrix
+    Matrix([
+    [ 0,  0, -1,  0, -1],
+    [ 0, -1,  0, -1,  0],
+    [-1,  0,  1,  0,  0],
+    [ 0, -1,  0,  0,  1],
+    [-1,  0,  0,  1,  0]])
+    >>> matrix.det()
+    0
+
+    Reference
+    ==========
+    1. [Kapur1994]_
+    2. [Palancz08]_
+
+    See Also
+    ========
+    Notebook in examples: sympy/example/notebooks.
+    """
+
+    def __init__(self, polynomials, variables):
+        """
+        A class that takes two lists, a list of polynomials and list of
+        variables. Returns the Dixon matrix of the multivariate system.
+
+        Parameters
+        ----------
+        polynomials : list of polynomials
+            A  list of m n-degree polynomials
+        variables: list
+            A list of all n variables
+        """
+        self.polynomials = polynomials
+        self.variables = variables
+
+        self.n = len(self.variables)
+        self.m = len(self.polynomials)
+
+        a = IndexedBase("alpha")
```

Comment:
It is also possible to define ordinary symbols that print nicely:
```
>>> pprint(symbols('alpha:5'))
(α₀, α₁, α₂, α₃, α₄)
```

Reply:
I hadn't realise that you could define a range of symbols the way you did, thus I used the `IndexedBase` class. Do you think that it makes a big difference? Should I replace the `IndexedBase` with the `symbols` class?

Reply:
I don't think it makes any difference in the operation. There may be some difference in the output.

BTW, the notebook names contain spaces. Those should be avoided. The names could be simply something like "Sylvester's_resultant.ipynb".